### ISSUE TITLE
Arcade is not setting the CacheControl to no-cache when uploading badges

### ISSUE THREAD

**Issue by livarcocc** [dotnet/arcade#3948]
With the switch to using PushToAzureDevOpsArtifacts to push the badges to blob storage, we noticed that the badges are showing up stale in the github.com/dotnet/core-sdk readme page.

This happens because the CacheControl for the badges is not being set and they are being cached by the browser.

When uploading these files, the CacheControl should be set to no-cache.

Is this something that the task supports already?

cc @JohnTortugo

**Comment by riarenas** [comment]
@JohnTortugo @mmitche Should we bring this back to release/3.x ? if this is causing stale badges in the repos?

**Comment by mmitche** [comment]
yes I think so

**Comment by JohnTortugo** [comment]
Yeah. Agree.

**Comment by RussKie** [comment]
https://github.com/dotnet/core-sdk still doesn't look correct:
![image](https://user-images.githubusercontent.com/4403806/66445528-d28b9a80-ea92-11e9-95bd-7fac4ea46fa5.png)

**Comment by JohnTortugo** [comment]
@MattGal perhaps this is something that FR can look?

**Comment by MattGal** [comment]
@JohnTortugo doesn't seem terribly urgent but yes seems reasonable for FR, took it.

**Comment by rokonec** [comment]
It turned out that GitHub has its own image proxy - [Camo](https://help.github.com/en/github/authenticating-to-github/about-anonymized-image-urls) - which reflects cache-control but was already initiated by images without cache-control. This cause Camo to return it with year-long cache control header. 
For example 
![](https://camo.githubusercontent.com/fb5279d439da8fda508b6997baa84202aacf1643/68747470733a2f2f646f746e6574636c692e626c6f622e636f72652e77696e646f77732e6e65742f646f746e65742f53646b2f6d61737465722f6c696e75785f7836345f52656c656173655f76657273696f6e5f62616467652e737667) 
`curl -I https://camo.githubusercontent.com/fb5279d439da8fda508b6997baa84202aacf1643/68747470733a2f2f646f746e6574636c692e626c6f622e636f72652e77696e646f77732e6e65742f646f746e65742f53646b2f6d61737465722f6c696e75785f7836345f52656c656173655f76657273696f6e5f62616467652e737667`
vs its corresponding 
![](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/master/linux_x64_Release_version_badge.svg?no-cache) 
`curl -I https://dotnetcli.blob.core.windows.net/dotnet/Sdk/master/linux_x64_Release_version_badge.svg`

There is a way to purge image from Camo proxy by `curl -X PURGE https://camo.githubusercontent.com/fb...` but for those unfortunate whose browser already cached previous Camo proxy response for a year, it would not make practical difference, unless they hit CTRL+F5. 

I would therefore recommend to modify related README.md content by simple find & replace from: `_version_badge.svg` to `_version_badge.svg?no-cache`

This will trigger github to compute and use different/new Camo proxy URL.

Please note that instead of ?no-cache almost anything will work for example ?nc or ?be-aware-of-scissors

I have tested it and it works on my machine :-)

@RussKie Is this solution acceptable for your team?

**Comment by RussKie** [comment]
> Is this solution acceptable for your team?

Yes, thank you. As long as our customers see correct labels, I'm happy.
👍
